charts/testkube-operator/values.yaml

```diff
@@ -45,8 +45,8 @@ webhook:
   patch:
     enabled: true
     image:
-      repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
-      version: v1.3.0
+      repository: dpejcev/kube-webhook-certgen
```

Comment:
is the image public?Otherwise, we'd need to add docker authentication to a cluster.
and maybe it's better to push it to `kubeshop` organization?

Reply:
it is my fork of kube-webhook-certgen and it is public.
Needs to be moved to kubeshop org, but I am not sure I have all privileges to setup CI and rest

Reply:
we have `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN` secrets that are added on Org level so they can be used in any CI repo for pushing images.